eureka-core/src/main/java/com/netflix/eureka/registry/AbstractInstanceRegistry.java

```diff
@@ -1315,21 +1238,4 @@ public void run() {
 
         };
     }
-
-    protected void initRemoteRegionRegistry() throws MalformedURLException {
```

Comment:
Remote region abstraction is parallel to PeerAwareInstanceRegistry, I would leave it here as we actually use the remote data in this class.